roads.mss

```diff
@@ -125,8 +131,11 @@
 @motorway-width-z15:             10;
 @motorway-link-width-z15:         7.8;
 @trunk-width-z15:                10;
+@trunk-link-width-z15:          7.8;
 @primary-width-z15:              10;
+@primary-link-width-z15:        7.8;
 @secondary-width-z15:             9;
+@secondary-link-width-z15:        7;
```

Comment:
Why 7 here instead of 7.8 as all the others?

Reply:
I just made it a bit smaller because secondary-with is also a bit smaller (9) than the other bigger road types above it (10).